src/runtime_src/core/pcie/driver/linux/xocl/subdev/qdma.c

```diff
@@ -28,7 +28,11 @@
 #include "../lib/libqdma/libqdma_export.h"
 #include "qdma_ioctl.h"
 
-#define XOCL_FILE_PAGE_OFFSET   0x100000
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(4, 17, 0))
```

Comment:
it might be better to use #ifndef here. centos may backport code from new kernel to their 3.10 kernel without bumping up the kernel version. so, the check against kernel version might not be accurate.

Reply:
OK, it is a good idea, i have get and fix it.